guides/README.md

```diff
@@ -17,7 +17,7 @@ Contents :
 
 First of all, you must have access to a running instance of MeiliSearch.
 
-There are [several download possibilities](/guides/advanced_guides/binary.md#download-and-launch).
+There are several download possibilities.
```

Comment:
Did you remove it because I copy pasted the possibilities underneath it ?
Because I thought about it, and even if it is redundant, the objective of the getting started page is to link to the other pages of the documentation. In this context, i'd prefer to leave the link. What do you think ?

Reply:
I think there are already too much things to click everywhere. And when you click you have exactly the same tabs than below. Quite frustrating. It looks like we forgot to remove it.
But it's as you want. I really don't like "there are a lot of links throughout the docs" when I read documentation, but I'm not everybody :) I really thought you forgot to remove it.

Reply:
I get your point but I would prefer keeping it. Thanks for the changes though !